packages/dev/core/src/Engines/AbstractEngine/abstractEngine.timeQuery.ts

```diff
@@ -0,0 +1,33 @@
+import { AbstractEngine } from "../abstractEngine";
+import { PerfCounter } from "../../Misc/perfCounter";
+import type { Nullable } from "../../types";
+
+declare module "../../Engines/abstractEngine" {
+    export interface AbstractEngine {
+        /** @internal */
+        _captureGPUFrameTime: boolean;

```

Comment:
Seems like this is an implementation details of captureGPUFrameTime. Since captureGPUFrameTime must be implemented by derived classes, it seems like derived classes should decide whether they want/need this property and declaring it.